plugins/funder.c

```diff
@@ -1593,6 +1593,17 @@ static char *amount_option(struct plugin *plugin, const char *arg, bool check_on
 	return NULL;
 }
 
+static bool jsonfmt_amount_sat(struct plugin *plugin,
+			       struct json_stream *js,
+			       const char *fieldname,
+			       struct amount_sat *sats)
+{
+	/* We do not expose raw numbers for sats fields: raw numbers
+	 * in our interface means MSAT! */
```

Comment:
Good call. It's one thing for the user to set these parameters in the config, but another thing to consume them here.  It might be nice if we could enforce a suffix/formatting on non-default configs as well.